docs/ga-roadmap.md

```diff
@@ -0,0 +1,172 @@
+
+At the end of this phase we want to achieve core APIs stability. This will allow developers
+to implement custom components and extend the collector will be marked as stable.
+The complete list of the packages/modules will be finalized during the first action item of
+this phase, but the tentative list is:
+
+* `consumer`
+  * Official internal data model `pdata`.
+  * Interfaces and utils to build a Consumer for (trace, metrics, logs).
+* `config`
+  * Core `config` including service definition, component definition will be stabilized.
+  * To be determined which config helpers will be marked as stable (e.g. configgrpc, etc.).
+* `component`
+  * Interfaces and utils to build a Collector component (receiver, processor, exporter, extension).
+* `obsreport`
+  * Focus on the public API of this package. It is out of scope to ensure stability for the
+  metrics emitted (focus in phase 2).
+* `service`
+  * Public API to construct a OpenTelemetry Collector Service.
+
+**Action Items:**
+
+* Create a new milestone for this phase, create issues for all the other action items and add
+them to the milestone.
+* Agreement on all packages/modules that will be marked as stable during this phase.
+* Write a version doc as per [version and stability document](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md).
+  * Previously it was discussed that for the Collector it is fine to release stable golang modules
+  that contain APIs marked as experimental.
+  * Define status schema (experimental/stable), what are they applicable to every module.
+* Investigate if splitting into smaller, more granular, modules is possible.
+  * Define the modules schema, try to not break everyone.
+  See [here](https://github.com/golang/go/wiki/Modules#is-it-possible-to-add-a-module-to-a-multi-module-repository).
+  * Investigate how can we release multiple golang modules from the same repo, without asking
+  people that consume them to use a replace statement. See problem in contrib.
+  * Investigate how to release test-only utils.
+* Review all public APIs and godocs for modules that we want to release in this phase.
+  * Fix all critical issues, and remove unnecessary (“When in doubt leave it out”) public APIs.
+  * Remove all already deprecated code from the stable modules.
+* Transition to opentelemetry-go trace library from opencensus?
+* Investigate if any config helper needs to be released as stable, if any do the review of
+the public API, godoc and configuration.
+* Investigate tools that check for API compatibility for go modules, enable them for modules
+that we mark as stable.
+
+### Phase 2
+
+**Tentative Date:** 2021-04-31
+
+**Key Results:** At the end of this phase the Collector’s end-to-end support for OTLP traces
+only be marked as GA.
+
+At the end of this phase we want to ensure that the Collector can be run in production, it can receive
```

Comment:
Do we need to define "can be run in production"? I think the goals of production-readiness could help double-check the AIs.